app/cni/pkg/cni/annotations.go

```diff
@@ -94,6 +97,20 @@ func validatePortList(ports string) error {
 	return nil
 }
 
+func validateBool(val string) error {
+	if val == "" {
+		return errors.New("value is empty")
+	}
+
+	validValues := []string{"yes", "no", "true", "false"}
```

Comment:
is this  the first time we're allowing "yes" for boolean? this should be consistent across all toggles and we shouldn't add it just here

Reply:
Yes, this can be the first time introducing "yes/no". Right, agreed, we'd better make it consistent across the product. I search over the whole product and found only "enabled/disabled" for things like enabling the sidecar injector. 
I personally think adding "yes/no" is more intuitive when setting boolean values. Here it is setting a boolean value rather than an "enabled state", in fact, the name of the switch here contains a "disable", so it will be misleading to reuse "enabled/disabled" in this case.

Reply:
Had a chat with Charly, I'll change the argument to `ipv6`, so we can reuse existing values like `enabled/disabled`